website/config/routes.js

```diff
@@ -34,6 +34,9 @@ module.exports.routes = {
 
   'GET /apply': 'https://fleet-device-management.breezy.hr',
 
+  'GET /documentation': { action: 'view-documentation' },
+  'GET /docs-template': { action: 'view-docs-template' },
```

Comment:
I think we should move these to `config/env/dev` and sandbox them for the moment so we can get this merged (see line 21 for the existing redirect conflict)